src/SshCert/Metadata.php

```diff
@@ -80,9 +80,13 @@ private function readString(&$bytes) {
      * @return int
      */
     private function readUint64(&$bytes) {
-        $ret = \unpack('J', \substr($bytes, 0, 8));
+        $packed = \substr($bytes, 0, 8);
         $bytes = \substr($bytes, 8);
```

Comment:
This line can be deleted, no?

Reply:
This is removing 8 bytes by reference from `$bytes`, so the decoding can go on to the next field.

Reply:
but `$bytes` isn't used after that from what I can see

Reply:
ah, by reference, my bad